src/renderer/src/routes/(MapTabs)/_Map.test.tsx

```diff
@@ -1,54 +1,94 @@
+import { describe } from 'node:test'
 import type { ReactNode } from 'react'
-import {
-	RouterProvider,
-	createRootRoute,
-	createRoute,
-	createRouter,
-} from '@tanstack/react-router'
+import { QueryClient } from '@tanstack/react-query'
+import { RouterProvider } from '@tanstack/react-router'
 import { render, screen } from '@testing-library/react'
 import { expect, test } from 'vitest'
 
-import { IntlProvider } from '../../contexts/IntlContext'
-import { MapLayout } from './_Map'
+import { router } from '../../App'
```

Comment:
similar to how we should be working with zustand stores in tests, I _think_ the tests should avoid using this singleton instance and instead create their own instance within the test to avoid potential issues related to shared instances across tests i.e.

```tsx
describe('tabs navigate to correct routes', () => {
	const router = createRouter({ routeTree })

	render(<RouterProvider router={router} />, { wrapper: Wrapper })

    // rest of tests...
```